OnPod/Node/Examples/Ethernet/ethernet.cpp

```diff
@@ -0,0 +1,48 @@
+#include <Arduino.h>
+#include <Ethernet.h>
+#include <EthernetUdp.h>
+#include "ethernet.h"
+
+UDPClass::UDPClass(uint8_t ETHERNET_SS_PIN, IPAddress NODE_IP,
+                    uint16_t NODE_PORT, uint8_t NODE_TYPE):
+    ETHERNET_SS_PIN(ETHERNET_SS_PIN),
+    NODE_IP(NODE_IP),
+    NODE_PORT(NODE_PORT),
+    NODE_TYPE(NODE_TYPE)
+    {}
+
+void UDPClass::init() {
```

Comment:
Is there a reason to separate `init()` and the constructor? Maybe it's just a style thing I'm not aware of, either way, it's not a huge deal

Reply:
The goal of this was to quickly instantiate all of the global variables without any possibility of error.

Then, later in the code, you can call the ‘UDPClass::init()’ function to take care of the more time-consuming calls. This gives you more control over the Node’s boot timing.

I think that makes sense? @stuckatmarine

Reply:
@stuckatmarine says I'm stupid and this is fine, don't mind me